src/libraries/System.Diagnostics.EventLog/src/Resources/Strings.resx

```diff
@@ -239,4 +239,10 @@
   <data name="LogAlreadyExistsAsSource" xml:space="preserve">
     <value>Log {0} has already been registered as a source on the local computer.</value>
   </data>
+  <data name="ArgumentOutOfRange_NeedNonNegOrNegative1" xml:space="preserve">
```

Comment:
why are edits to eventlog needed -- I don't see what this library pulls in that now needs it?

Reply:
I think I got that wrong.